sormas-backend/src/test/java/de/symeda/sormas/backend/externalmessage/TestReportFacadeEjbTest.java

```diff
@@ -1,31 +1,32 @@
-package de.symeda.sormas.backend.labmessage;
+package de.symeda.sormas.backend.externalmessage;
```

Comment:
If the tested class gets moved (based on a review comment), this one needs to be moved as well.